modules/eks/cluster/CHANGELOG.md

```diff
@@ -1,11 +1,211 @@
-## Components PR [#910](https://github.com/cloudposse/terraform-aws-components/pull/910)
+## Breaking Changes: Components PR [#1033](https://github.com/cloudposse/terraform-aws-components/pull/1033)
+
+### Major Breaking Changes
+
+:::warning Major Breaking Changes, Manual Intervention Required
+
+This release includes a major breaking change that requires manual intervention to migrate existing clusters. The change
+is necessary to support the new AWS Access Control API, which is more secure and more reliable than the old `aws-auth`
+ConfigMap.
+
+:::
+
+This release drops support for the `aws-auth` ConfigMap and switches to managing access control with the new AWS Access
+Control API. This change allows for more secure and reliable access control, and removes the requirement that Terraform
+operations on the EKS cluster itself require network access to the EKS control plane.
+
+In this release, this component only supports assigning "team roles" to Kubernetes RBAC groups. Support for AWS EKS
+Access Policies is not yet implemented. However, if you specify `system:masters` as a group, that will be translated
+into assigning the `AmazonEKSClusterAdminPolicy` to the role. Any other `system:*` group will cause an error.
+
+:::tip Network Access Considerations
+
+Previously, this component required network access to the EKS control plane to manage the `aws-auth` ConfigMap. This
+meant having the EKS control plane accessible from the public internet, or using a bastion host or VPN to access the
+control plane. With the new AWS Access Control API, Terraform operations on the EKS cluster no longer require network
+access to the EKS control plane.
+
+This may seem like it makes it easier to secure the EKS control plane, but Terraform users will still require network
+access to the EKS control plane to manage any deployments or other Kubernetes resources in the cluster. This means that
+this upgrade does not substantially change the need for network access.
+
+:::
+
+### Minor Changes
+
+With the fixes included and AWS Terraform Provider v5.43.0 and Karpenter v0.33.0, the
+`legacy_do_not_create_karpenter_instance_profile` is now obsolete. After upgrading both this component and the
+`eks/karpenter` component, if you had it in your configuration, you can remove it. If you had previously set it to
+`false`, removing it may cause an error when you apply the changes. If you see an error about the
+`aws_iam_instance_profile` resource being destroyed (cannot be destroyed because it is in use, has dependencies, and/or
+has role attached), you can simply remove the resource from the Terraform state with `[atmos] terraform state rm`,
+because it will be managed by the Karpenter controller instead of Terraform.
+
+### Migration Procedure
```

Comment:
Assuming this file grows arbitrarily in the future, assume there could be a collision with this title:
```suggestion
### Access Control API Migration Procedure
```